<launch>

    <group>
        <!-- argumentos -->
        <arg name="material" default="Plastic"/>
        <arg name="show_original" default="False"/>
        <arg name="debug" default="False"/>

        <node pkg="es_vision" name="get_video" type="get_video" output="screen"/>
        <node pkg="es_vision" name="segment" type="segment" output="screen"/>

        <!-- Parametros -->
        <param name="/material" value="$(arg material)" />
        <param name="/show_original" value="$(arg show_original)" />

        <group if="$(eval arg('debug') ==True)">
            <node pkg="es_vision" type="tune" name="tune" output="screen"/>
            <param name="/show_original" value="True" />
        </group>

        <param name="/debug" value="$(arg debug)" />
        <param name="/H_MAX" value="255" />
        <param name="/H_MIN" value="0" />
        <param name="/S_MAX" value="255" />
        <param name="/S_MIN" value="0" />
        <param name="/V_MAX" value="255" />
        <param name="/V_MIN" value="0" />
    </group>
    
</launch>